app/models/new.rb

```diff
@@ -0,0 +1,5 @@
+class New < ActiveRecord::Base
```

Comment:
why class name is New? this name has pontential risk to mixed with some `system` names. Change it on difference